### PR TITLE
[LOC]MWPW-163466:  Integrate Project Update API with "publish" flag when user clicks "Create Project" in Actions Screen

### PR DESCRIPTION
**Pull request by saurabhsircar11**
* Integrated Publish call
* Pre populate the Name field in case of translate and publish
* After project is created, pressing on view will take you to the project details screen of loc

Resolves: https://jira.corp.adobe.com/browse/MWPW-163466

**Test URLs:**
- Before: https://main--milo--adobecom.aem.page/drafts/sircar/locui-create?martech=off
- After: https://mwpw-163466-update-publish-project--milo--saurabhsircar11.hlx.page/drafts/sircar/locui-create?martech=off

### CODE REVIEW

File: libs/blocks/locui-create/input-actions/modal.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ function Modal() {
       <strong>Project "${project.value.name || 'n/a'}" Successfully Created</strong>
     </div>
     <div class="create-project-view">
-     <button class="s2-btn accent">View</button>
+    ${projectInfo.value.projectLink && html`<a class="s2-btn accent" href="${projectInfo.value.projectLink}" target="_blank" rel="noreferrer noopener">View</a>`}
```
Comment (nkthakur48) [inline libs/blocks/locui-create/input-actions/modal.js:12]: You may want to use `projectInfo?.value?.projectLink`

File: libs/blocks/locui-create/input-locale/index.js [hunk 2]
```diff
@@ -146,11 +151,42 @@ export default function useInputLocale() {
 
   const errorPresent = () => Object.keys(activeLocales).length > 0;
 
-  const handleNext = () => {
+  const prefillActionAndWorkflow = (languages) => {
+    const storedLanguages = project.value?.languages ?? [];
+    if (storedLanguages.length < 1) {
+      return languages.map((lang) => ({ ...lang, action: project.value.type === LOCALIZATION_TYPES.translation ? 'Translate' : 'Rollout', workflow: '' }));
+    }
+    let i = 0;
```
Comment (nkthakur48) [inline libs/blocks/locui-create/input-locale/index.js:159]: Maybe use a better variable name than `i`

File: libs/blocks/locui-create/input-urls/index.js [hunk 3]
```diff
@@ -84,3 +85,12 @@ export async function findFragments(urls) {
   const foundFragments = Object.values(fragmentsByPathname);
   return validateUrlsFormat(foundFragments, true);
 }
+export function getInitialName(type) {
+  const preffix = type === LOCALIZATION_TYPES.rollout ? 'rollout' : 'translate';
```
Comment (nkthakur48) [inline libs/blocks/locui-create/input-urls/index.js:89]: Typo in variable name `prefix`

File: libs/blocks/locui-create/utils/constant.js [hunk 4]
```diff
@@ -1,6 +1,11 @@
 export const LOCALES = 'locales';
 export const LOCALE_GROUPS = 'localegroups';
 export const URL_SEPARATOR_PATTERN = /[,\r\n]/g;
+export const API_BASE_URL = {
+  dev: 'https://14257-miloc-dev.adobeioruntime.net/api/v1/web/miloc-0.0.1',
```
Comment (nkthakur48) [inline libs/blocks/locui-create/utils/constant.js:5]: @raga-adbe-gh Could you please get these added in Milo config file as well? Once done, we will remove it and pick it from there

**Review by nkthakur48 — CHANGES_REQUESTED** [review]
@saurabhsircar11 Added few minor comments. Please have a look

File: libs/blocks/locui-create/input-actions/modal.js (hunk 1 above)
Comment (saurabhsircar11) [reply libs/blocks/locui-create/input-actions/modal.js:12]: Hi @nkthakur48 , 
I checked with @raga-adbe-gh on this, he said that we need to use project link to go to the project details screen, as the preview url is not available at that time.

File: libs/blocks/locui-create/input-locale/index.js (hunk 2 above)
Comment (saurabhsircar11) [reply libs/blocks/locui-create/input-locale/index.js]: Thanks. Fixed the comment

File: libs/blocks/locui-create/input-urls/index.js (hunk 3 above)
Comment (saurabhsircar11) [reply libs/blocks/locui-create/input-urls/index.js]: Thanks. Fixed

**Review by nkthakur48 — APPROVED** [review]

**Review by hnvadobe — APPROVED** [review]

**Comment by nkthakur48** [comment]
Thanks @saurabhsircar11  and @Zweihand3r for working on the changes

**Pull request merged**